src/app.tsx

```diff
@@ -48,11 +48,17 @@ export interface FolderFileInfo extends FileInfo {
     to: string | null,
 }
 
+interface FilesContextType {
+    addAlert: (title: string, variant: AlertVariant, key: string) => void,
+    cwdInfo: FileInfo | null,
+}
+
 export const FilesContext = React.createContext({
-    // eslint-disable-next-line no-unused-vars
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    addAlert: (title: string, variant: AlertVariant, key: string) => console.warn("FilesContext not initialized")
-});
+    addAlert: () => console.warn("FilesContext not initialized"),
```

Comment:
I have a very strong déjà vu on this one. Dropping the signature here feels wrong at least for documentation purposes, but also for type checking -- this is very much not a `FilesContextType` now, but really should be. Perhaps prefix the parameter names with `_`, does that work? If not, I'd prefer keeping the eslint overrides.

Reply:
Yeah it's about the unused warnings which are super annoying.

Reply:
But honestly, this is valid JS, it is a valid `addAlert` function as one can check `arguments` and calling addAlert with extra arguments is fine in JavaScript.

I can add it, and add `    // eslint-disable-next-line @typescript-eslint/no-unused-vars` but it feels wrong. The Object is typed as FilesContextType and this still works

![image](https://github.com/cockpit-project/cockpit-files/assets/67428/685ca466-0eba-436c-94ae-e86c79292df5)

Reply:
Well, I'm not very familiar with type checking, so if you say that's kosher, then so be it. :grin: